apps/frontend/app/components/dashboard/forms/metadata-progress-update/utils/common-elements.tsx

```
@@ -0,0 +1,105 @@
+			onChange={(v) => onChange(v as WatchTimes)}
+		/>
+	);
+};
+
+interface CustomDatePickerProps {
+	selectedDate: Date | null;
+	onDateChange: (date: Date | null) => void;
+}
+
+export const CustomDatePicker = ({
+	selectedDate,
```

Biome (1.9.4)

[error] This variable is unused.

Unused variables usually are result of incomplete refactoring, typos and other source of bugs.

(lint/correctness/noUnusedVariables)